packages/dioxus/Cargo.toml

```diff
@@ -58,6 +58,7 @@ ssr = ["dep:dioxus-ssr", "dioxus-router?/ssr", "dioxus-config-macro/ssr"]
 liveview = ["dep:dioxus-liveview", "dioxus-config-macro/liveview", "dioxus-router?/liveview"]
 static-generation = ["dep:dioxus-static-site-generation", "dioxus-config-macro/static-generation"]
 axum = ["dioxus-fullstack?/axum", "dioxus-fullstack?/server", "dioxus-static-site-generation?/server", "ssr", "dioxus-liveview?/axum", "dep:axum"]
+server = ["axum"]
```

Comment:
does it make sense to still have `axum` as a feature name here instead of just renaming it? Or we're trying to retain backwards compat?

Reply:
It is just there for backwards compatibility. I'm not sure if we can mark a feature as deprecated? Maybe through a build script?